ui/library/less/text-layer.less

```diff
@@ -0,0 +1,82 @@
+
+
+// this was taken directly from the react-pdf file at:  'react-pdf/dist/esm/Page/TextLayer.css';
```

Comment:
@ericmarsh995 can you make a ticket and put a ref: of that ticket in here too so we won't forget it